scratch-render/rspack.config.js

```
@@ -0,0 +1,105 @@
+                                },
+                            },
+                        ],
+                    ],
+                },
+            },
+        ],
+    },
+    optimization: {
+        minimizer: [
+            new rspack.SwcJsMinimizerRspackPlugin({
+                minimizerOptions: { mangle: false },
```

ESLint

[error] There should be no space after '{'.

(object-curly-spacing)